test/ruby/test_compile_prism.rb

```diff
@@ -589,6 +589,17 @@ def test_InterpolatedStringNode
       assert_prism_eval('$pit = 1; "1 #$pit 1"')
       assert_prism_eval('"1 #{1 + 2} 1"')
       assert_prism_eval('"Prism" "::" "TestCompilePrism"')
+      assert_prism_eval('("a""b").frozen?')
+      assert_prism_eval(<<-CODE)
+        # frozen_string_literal: true
+
+        ("a""b").frozen?
+      CODE
+      assert_prism_eval(<<-CODE)
+        # frozen_string_literal: true
+
+        ("a""b""#{1}").frozen?
+      CODE
```

Comment:
Can we add a test case to exercise the `putobject`/`putstring` that happens after the loop if we still have items in the `current_string`? 

That is:
```suggestion
      CODE
      assert_prism_eval(<<-CODE)
        # frozen_string_literal: true
        
        ("a""#{1}""b").frozen?
      CODE
```